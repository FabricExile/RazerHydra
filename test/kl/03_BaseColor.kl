require RazerHydra;

operator entry() {
  RazerHydraController controller();
  UInt8 red, green, blue;

  controller.setBaseColor(255, 0, 0);
  controller.getBaseColor(red, green, blue);
  report('red '+red+', green '+green+', blue '+blue);

  controller.setBaseColor(0, 255, 0);
  controller.getBaseColor(red, green, blue);
  report('red '+red+', green '+green+', blue '+blue);

  controller.setBaseColor(0, 0, 255);
  controller.getBaseColor(red, green, blue);
  report('red '+red+', green '+green+', blue '+blue);
}