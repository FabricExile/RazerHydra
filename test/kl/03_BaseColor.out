[FABRIC:MT] RazerHydra: Controller initialized.
[FABRIC:MT] red 255, green 0, blue 0
[FABRIC:MT] red 0, green 255, blue 0
[FABRIC:MT] red 0, green 0, blue 255
[FABRIC:MT] RazerHydra: Controller shutdown.